packages/identifier/src/handle.ts

```diff
@@ -139,15 +141,23 @@ export const ensureHandleServiceConstraints = (
   if (reserved[front]) {
     throw new ReservedHandleError('Reserved handle')
   }
+  
+  for (const bannedWord of banned) {
+    const regex = new RegExp(`.*${bannedWord}.*`)
```

Comment:
does the regex need to be case insensitive?
```suggestion
    const regex = new RegExp(`.*${bannedWord}.*`, "i")
```

Reply:
Looks like it’s handled here: https://github.com/bluesky-social/atproto/blob/0bac4e775242b1916ed4216ab113b9553facf8fa/packages/identifier/src/handle.ts#L90